lib/scoped_search/query_builder.rb

```diff
@@ -485,6 +489,16 @@ def to_sql(builder, definition, &block)
             raise ScopedSearch::QueryNotSupported, "Don't know how to handle this operator node: #{operator.inspect} with #{children.inspect}!"
           end
         end
+
+        private
+
+        def validate_value(field, value)
+          validator = field.validator
+          if field.validator && field.only_explicit
+            valid = validator.call(value)
```

Comment:
Right now it would be considered as validation failure.
But probably it should be considered as a success, so the validator could look either like this (in simple boolean case):

``` ruby
->(value) { value ~= /my_valid_regex/ }
```

or in case it wants to customize the error:

``` ruby
->(value) { "My failure message" if value ~= /my_valid_regex/ }
```

Does it sound like a good user/coder experience?